spec/controllers/chapters_controller_spec.rb

```diff
@@ -195,6 +195,26 @@
       expect(assigns[:page_title]).to eq("page title")
     end
 
+    context "when multi-chapter work is missing tags" do
+      let(:chapter1) { work.chapters.first }
+      let(:chapter2) { create(:chapter, work: work, posted: true, position: 2) }
+
+      before do
+        work.taggings.delete_all
+        work.save
+        work.reload
+        fake_login_known_user(user)
+      end
+
+      it "can still render chapter by chapter" do
+        get :show, params: { work_id: work.id, id: chapter1 }
+        expect(response).to have_http_status(:ok)
+        expect(assigns(:page_title)).to include(assigns(:work).title)
+        expect(assigns(:page_title)).to include("No fandom specified")
+        expect(assigns(:page_title)).to include(" - Chapter 1")
+      end
+    end
+
```

Comment:
Huh, yeah, I'm not clear on why it wasn't working for logged out users, either. I wanted to blame `get_page_title`, but I didn't see an obvious issue there.

Regardless, this should work instead -- it's shorter and more in line with the other  `@page_title` tests:
```suggestion
    context "when work has no fandom" do
      it "assigns @page_title with a placeholder for the fandom" do
        allow_any_instance_of(Work).to receive(:tag_groups).and_return("Fandom" => [])
        expect_any_instance_of(ChaptersController).to receive(:get_page_title).with("No fandom specified", user.pseuds.first.name, "#{work.title} - Chapter 1").and_return("page title")
        get :show, params: { work_id: work.id, id: work.chapters.first.id }
        expect(response).to have_http_status(:ok)
        expect(assigns[:page_title]).to eq("page title")
      end
    end

```

Reply:
Maybe it's because deleting all the taggings causes `@work.adult?` to evaluate to true (it checks `@work.ratings.blank?`), so it shows the adult content warning instead of calculating the page title ([AO3-5502](https://otwarchive.atlassian.net/browse/AO3-5502)). When logged in as the work's creator, it automatically skips the adult content warning.

Reply:
Ah good point.  I think that's it.  Technically we could get by with just deleting the fandom, but I thought it would be good to cover the case where all the required tags are missing.  Regardless I'll incorporate @sarken's suggestion.